_pages/telco/tua-pre-release-information.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Pre-Release Information
+---
+
+This document describes what is included in the latest pre-release of TUA Spartacus libraries, such as `next` and `rc` libraries.
+
+_Last updated July 24, 2020 by Deborah Cholmeley-Jones, Solution Owner, TUA Spartacus_
+
+## Release 1.0.0-next.2 - July 24th, 2020
+
+Aside from continued work on removing deprecated code, bug fixes, and code refactoring, the following sections highlight work that has been done for this pre-release.
+
+- Product Offering Pricing improvements. For more information, see [Pricing in TUA Help Portal](https://help.sap.com/viewer/32f0086927f44c9ab1199f1dab8833cd/2007/en-US/ad4430d10fc3477096752d83f935faf9.html) in the TUA Help portal.
```

Comment:
Please update "Product Offering Pricing improvements" to "Product Offering pricing enhancements (display of all one-time charges, recurring charges and usage charges).

Reply:
The suggested change is incorporated in the updated topic in the attached zip for confirmation. 
[Release Information for TUA - Spartacus Documentation.zip](https://github.com/SAP/spartacus-docs/files/4982085/Release.Information.for.TUA.-.Spartacus.Documentation.zip)